common/api/core-frontend.api.md

```diff
@@ -2074,6 +2075,13 @@ export enum ContextRotationId {
     View = 6
 }
 
+// @beta

```

Comment:
Perhaps consider a NextVersion.md entry.

Reply:
I was about to add one but I see master is now targeting 5.1.0. This will need to be backported to 5.0.0, so any changelogs belong on that branch's 5.0.0.md, not in NextVersion.md.

Reply:
Sounds good.